### ISSUE TITLE
Panic with `query web` plugin

### ISSUE THREAD

**Issue by mhnap** [nushell/nushell#11529]
### Describe the bug

I ran `http get https://en.wikipedia.org/wiki/List_of_cities_in_India_by_population | query web` and got panic: 
```
thread 'main' panicked at /home/mhnap/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nu_plugin_query-0.89.0/src/query_web.rs:268:42:
Error unwrapping scraperselector::parse: UnexpectedSelectorParseError(EmptySelector)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error:   × Unable to decode call for query web
   ╭─[entry #21:1:1]
 1 │ http get https://en.wikipedia.org/wiki/List_of_cities_in_India_by_population | query web
   ·                                                                                ────┬────
   ·                                                                                    ╰── Plugin failed to encode: EOF while parsing a value at line 1 column 0
   ╰────
```

### How to reproduce

I ran the following commands: 
1. `cargo install nu_plugin_query`
2. `register /home/mhnap/.cargo/bin/nu_plugin_query`
3. `http get https://en.wikipedia.org/wiki/List_of_cities_in_India_by_population | query web`

### Expected behavior

No panic.

### Screenshots

![image](https://github.com/nushell/nushell/assets/17296656/4d8d7cc2-deff-4ab1-8f3d-c842ec2a4c54)

### Configuration

| key                | value                                                   |
| ------------------ | ------------------------------------------------------- |
| version            | 0.89.0                                                  |
| branch             |                                                         |
| commit_hash        | 2c1560e281d65a50d90b30c42006755b364b4928                |
| build_os           | linux-x86_64                                            |
| build_target       | x86_64-unknown-linux-gnu                                |
| rust_version       | rustc 1.73.0 (cc66ad468 2023-10-03)                     |
| rust_channel       | 1.73.0-x86_64-unknown-linux-gnu                         |
| cargo_version      | cargo 1.73.0 (9c4383fb5 2023-08-26)                     |
| build_time         | 2024-01-09 20:27:07 +00:00                              |
| build_rust_channel | release                                                 |
| allocator          | mimalloc                                                |
| features           | default, sqlite, static-link-openssl, trash, which, zip |
| installed_plugins  | query, query json, query web, query xml                 |

**Comment by fdncred** [comment]
I agree that it shouldn't panic but piping into query web without any parameters isn't something people should do. query web -t fails with a stack overflow sometimes for some reason. not sure what's going on. query web needs a good overhaul.

**Comment by hustcer** [comment]
It won't panic for 0.109.1:
```console
Error: nu::shell::error

  × CSS query parse error
   ╭─[entry #6:1:80]
 1 │ http get https://en.wikipedia.org/wiki/List_of_cities_in_India_by_population | query web
   ·                                                                                ────┬────
   ·                                                                                    ╰─┤ Unexpected error occurred. Please report this to the developer
   ·                                                                                      │ EmptySelector
   ╰────
  help: cannot parse query as a valid CSS selector
```
